src/svelte5.js

```diff
@@ -0,0 +1,145 @@
+import {
```

Comment:
This feels like a lot of duplication for little payoff. If you want to have separate entry points, would it be better to re-export `pure.js`'s exports and only override the public methods that changed?

Reply:
My development steps are

1. Make it
2. Make it work
3. Make it pretty

This PR is at stage (2). all the functions that aren't different in `svelte5.js` should really be imported from `pure.js`. But first I was interested to see how deep the hole had to be. And then backfill and tidy up the garden again.

Reply:
Sure thing, happy to defer to whatever approach you want to take. Just wanted to point out opportunities nip stuff in the bud LoC-wise